spec/lib/config_spec.rb

```diff
@@ -14,20 +14,12 @@
     end
   end
 
-  describe 'enable_orm' do
+  describe 'setup_orm_adapter' do
     it 'adds specific error message to NameError exception' do
       expect do
         Doorkeeper.configure { orm 'hibernate' }
       end.to raise_error(NameError, /ORM adapter not found \(hibernate\)/)
     end
-
-    it 'does not change other exceptions' do
```

Comment:
True. This test should continue to exist though. For example, if we raise another exception due to a typo while developing, the exception should not be trapped but raised.